src/util/email.ts

```diff
@@ -0,0 +1,73 @@
+// Load Environment Variables
+import dotenv from 'dotenv';
+dotenv.config();
+
+import Email from 'email-templates';
+import { createTransport } from 'nodemailer';
+import SMTPTransport from 'nodemailer/lib/smtp-transport';
+import path from 'path';
+
+const getTransportOptions = (): SMTPTransport.Options | undefined => {
+  //With SMTP host/port we assume local development mailing
+  if (process.env.EMAIL_CONFIG_USERNAME && process.env.EMAIL_CONFIG_PASSWORD) {
+    return {
+      name: 'CubeCobra.com',
+      secure: true,
+      service: 'Gmail',
+      auth: {
+        user: process.env.EMAIL_CONFIG_USERNAME,
+        pass: process.env.EMAIL_CONFIG_PASSWORD,
+      },
+    };
+  } else if (process.env.EMAIL_CONFIG_SMTP_HOST && process.env.EMAIL_CONFIG_SMTP_PORT) {
+    return {
+      name: 'CubeCobra.com',
+      secure: false,
+      host: process.env.EMAIL_CONFIG_SMTP_HOST,
+      port: Number(process.env.EMAIL_CONFIG_SMTP_PORT),
+    };
+  } else {
+    return undefined;
+  }
+};
+
+const getTransport = () => {
+  const options = getTransportOptions();
+  return options ? createTransport(options) : undefined;
+};
+
+export const sendEmail = async (
+  to: string,
+  subject: string,
+  templateName: string,
+  templateLocals: Record<string, any> = {},
+): Promise<any> => {
+  const transport = getTransport();
+  if (!transport) {
+    // eslint-disable-next-line no-console -- Warn so clear why an email isn't being sent
+    console.warn('No email transport is configured, skipping sending');
```

Comment:
Given the developer setup instructions don't include how to setup gmail for emails to start, the default behaviour meant sending emails just caused server errors. Rather than do that, might as well be graceful and not error.